### PR TITLE
Controller refactor and first bulma work

### PR DESCRIPTION
**Pull request by CorneliaKelinske**
Hello Alan,

Can you let me know how to change the font color of my navbar text into black?

### CODE REVIEW

**Comment by alanvardy** [comment]
I added some css (`.nostyle`) that you can see here:

https://github.com/alanvardy/alan_vardy/blob/0b899bb1fb3145c4b73ea0f29123a599ed7ee298/assets/css/_components.scss#L114

And you can see one of the links I used it on here:

https://github.com/alanvardy/alan_vardy/blob/0b899bb1fb3145c4b73ea0f29123a599ed7ee298/lib/alan_vardy_web/templates/layout/_footer.html.eex#L5

File: lib/first_blog_web/controllers/page_controller.ex [hunk 1]
```diff
@@ -4,4 +4,12 @@ defmodule FirstBlogWeb.PageController do
   def index(conn, _params) do
     render(conn, "index.html")
   end
+
+  def blog(conn, _oarams) do
```
Comment (alanvardy) [inline lib/first_blog_web/controllers/page_controller.ex:8]: _params

File: lib/first_blog_web/router.ex [hunk 2]
```diff
@@ -16,9 +16,10 @@ defmodule FirstBlogWeb.Router do
   scope "/", FirstBlogWeb do
     pipe_through :browser
 
-    get "/blog", BlogController, :blog
-    get "/about", AboutController, :about_me
+    get "/blog", PageController, :blog
+    get "/about", PageController, :about_me
     get "/", PageController, :index
+    get "/home", PageController, :index
```
Comment (alanvardy) [inline lib/first_blog_web/router.ex:22]: nice

File: lib/first_blog_web/templates/layout/app.html.eex [hunk 3]
```diff
@@ -5,16 +5,23 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>FirstBlog · Phoenix Framework</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
```
Comment (alanvardy) [inline lib/first_blog_web/templates/layout/app.html.eex:8]: Cool, I love fontawesome. Icons go a long way.

**Review by alanvardy — APPROVED** [review]

**Comment by CorneliaKelinske** [comment]
Thanks for reviewing this.
I will look into the .nostyle option. I'm assuming that this will go into my custom.css file.

**Pull request merged**